src/stargate.rs

```diff
@@ -0,0 +1,46 @@
+use crate::{AcceptingModule, FailingModule, Module};
+use cosmwasm_std::{Binary, CosmosMsg, Empty, QueryRequest};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct StargateMsg {
+    pub type_url: String,
+    pub value: Binary,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct StargateQuery {
+    pub path: String,
+    pub data: Binary,
+}
+
+pub trait Stargate: Module<ExecT = StargateMsg, QueryT = StargateQuery, SudoT = Empty> {}
+
+pub type StargateAcceptingModule = AcceptingModule<StargateMsg, StargateQuery, Empty>;
```

Comment:
Interesting approach. Stargate messages and queries are not really modules. But this seems to work fine with the current model.

When renaming "stargate" -> "any" (see e.g. https://github.com/CosmWasm/cosmwasm/pull/1926) we can add an `AnyMsg` right in the CosmosMsg enum case instead of having the fields in the enum directly to make this easier for multitest.